src/models.py

```diff
@@ -1110,6 +1122,40 @@ def _lm_forward(self, batch, task, predict):
             pass
         return out
 
+    def _mc_forward(self, batch, task, predict):
+        ''' Forward for a multiple choice question answering task '''
```

Comment:
How does this handle the context vs. question distinction? Are they just concatenated?

Reply:
Yea they're concatenated during preprocessing. The question and the answer choice get the same segment ID, which seems standard for BERT.